README.md

```diff
@@ -1 +1,17 @@
-# multi-cluster-eval
\ No newline at end of file
+# Multi-cluster LLM Evals
+
+## Currently supported clusters
+- LEONARDO
+- LUMI
+- JURECA
+
+## Usage
+> Make sure that your HF_TOKEN is in your environment, otherwise you may be rated limited when pulling new evaluation datasets
+
+1. Clone this repository
+2. `cd multi-cluster-eval`
+3. `source cluster_env.sh` -> sets all relevant environment variables and activates the python environment
+4. Start an interactive session, e.g., `srun -p <partition-name> -t <time-limit> --gres="gpu:1" --pty <your-shell e.g., /bin/bash>`
```

Comment:
why not using the partition name and time limit from an env variable here? Also would be nice to have a line that can be copy-pasted:

```suggestion
3. Start an interactive session, e.g., `srun -p $default_partition -t $time_limit --gres="gpu:$num_gpu_per_node" --pty /bin/bash>`
```